lib/tasks/npq/bulk_change_applications_to_pending.rake

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+require "rake"
+
+namespace :npq_applications do
+  desc "Change NPQ applications to pending"
+  task :bulk_change_to_pending, %i[file dry_run] => :environment do |_task, args|
+    logger = Logger.new($stdout)
+    csv_file_path = args[:file]
+    dry_run = args[:dry_run] != "false"
+
+    unless File.exist?(csv_file_path)
+      logger.error "File not found: #{csv_file_path}"
```

Comment:
maybe worth adding some minimal docs for this - I'm being annoying hehe as this will be used by another team